From e70e8e91f97496a7df3140dd061b1a48b73d3251 Mon Sep 17 00:00:00 2001
From: Marsell Kukuljevic <marsell@joyent.com>
Date: Thu, 11 Oct 2018 07:55:15 +1300
Subject: [PATCH] TRITON-800 "Bhyve Max Vcpus" should be ignored if "Bhyve
 Capable" is not true Reviewed by: Trent Mick <trentm@gmail.com> Approved by:
 Trent Mick <trentm@gmail.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 27b790d..c764497 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "assert-plus": "~1.0.0",
     "bunyan": "1.8.5",
     "libuuid": "0.2.1",
-    "joyent-schemas": "git+https://github.com/joyent/schemas.git#9f0791091868f449c8933902b556652e6c24875c",
+    "joyent-schemas": "git+https://github.com/joyent/schemas.git#09cebfca94169601568fec60f7a6144130f7fd31",
     "jsprim": "1.3.1",
     "vasync": "~1.6.4"
   },
-- 
2.21.0

